trieste/utils/objectives.py

```diff
@@ -27,6 +27,7 @@
 from ..data import Dataset
 from ..observer import MultiObserver, Observer, SingleObserver
 from ..type import TensorType
+from ..space import Box
```

Comment:
heh, this seems to have created a circular import issue : | I hate these... 

it is logical to have these search space definitions in this file, alongside the objectives and minimizers, but then it will be tricky to get rid of this issue - suggestions anyone?

Reply:
One possible fix would be to stop utils importing objectives in its init.py file, since space only depends on shapes_equal from utils.misc. Objectives would then need to be explicitly imported as utils.objectives (just like multi_objectives already has to).

Reply:
Though this does point to the utils directory being used for two different things: general utility functions used in the actual code, and toy utility function for experimentation and writing tests.

Reply:
> Though this does point to the utils directory being used for two different things: general utility functions used in the actual code, and toy utility function for experimentation and writing tests.

how about moving the objectives to a new root-level package?

Reply:
I think this is probably the best solution, smth like `trieste/example_objectives/` or simply `trieste/objectives/`